app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/attachments/modals/attachmentImageModal.jsx

```diff
@@ -35,7 +35,7 @@ export class AttachmentImageModal extends Component {
   generateRotationCommand = (amount) => `rotate(${amount}deg)`;
 
   renderCustomButton = () => (
-    <div onClick={this.rotateImageHandler} left className={cx('attachment-image-rotate')}>
+    <div onClick={this.rotateImageHandler} left="true" className={cx('attachment-image-rotate')}>
```

Comment:
To avoid console error (see the screenshot below).
![image](https://user-images.githubusercontent.com/31861805/51826979-68163e00-22f9-11e9-989b-7c87bd332af6.png)